recipes/mamba-press/recipe.yaml

```diff
@@ -0,0 +1,58 @@
+context:
+  name: mamba-press
+  version: "0.0.1"
+  python_min: "3.13"
```

Comment:
seems like upstream thinks this is 3.12 min
https://github.com/mamba-org/mamba-press/blob/main/pyproject.toml#L53

which is correct?

Reply:
Seems like 3.12 is enough